dev/dll/Density/Compact.xaml

```diff
@@ -0,0 +1,40 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary 
+  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" 
+  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Default">
+            <x:Double x:Key="TextControlThemeMinHeight">24</x:Double>
```

Comment:
I wish these didn't need to be defined 3 times...

Reply:
I have no idea why It fail if I don't do it. I create #368 to track it.